linker/Linker/Driver.cs

```diff
@@ -72,7 +74,65 @@ public static int Execute (string[] args, ILogger customLogger = null)
 
 		public Driver (string [] args)
 		{
-			_queue = new Queue<string> (args);
+			_queue = ProcessResponseFile (args);
+		}
+
+		Queue<String> ProcessResponseFile (string [] args)
+		{
+			const char NUL = '\0';
+			var result = new Queue<string> ();
+			foreach (string arg in args) {
+				if (arg.StartsWith ("@")) {
+					string responseFileName = arg.Substring (1);
+					try {
+						IEnumerable<string> responseFileLines = File.ReadAllLines (responseFileName).Select (line => line.Trim ());
+						string responseFileText = string.Join (" ", responseFileLines) + NUL;
```

Comment:
I'd rather see length based parsing rather then tacking on null character and then assuming the contents of the arguments have no null characters.

Reply:
Fixed